### ISSUE TITLE
Rebase onto latest node-ssdp and take new logging into use

### ISSUE THREAD

**Issue by vjrantal** [thaliproject/Thali_CordovaPlugin#489]

**Comment by yaronyg** [comment]
@vjrantal Is this something you think we need to do for the March release? Because right now this bug is just in icebox so it won't be seen for a long time.

**Comment by vjrantal** [comment]
@yaronyg: I think this can be in icebox unless something comes up and we definitely need the detailed node-ssdp logging (or something else from the upstream project).

**Comment by yaronyg** [comment]
Agreed but boy do we want to do this as soon as we finish the new infra milestone since we will need it for debugging live releases.
